pkgs/by-name/gs/gssg/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  fetchFromGitea,
+  buildGoModule,
+}:
+
+buildGoModule {
+  pname = "gssg";
+  version = "0-unstable-2024-11-23";
+
+  src = fetchFromGitea {
```

Comment:
I'm still new to building nix packages, but wouldn't `fetchFromSourcehut` be better?

Example: https://git.sr.ht/~vendion/vendionOS/tree/master/item/pkgs/gssg/default.nix

Reply:
I'm still new too, I missed it when combing through fetchers, thanks